bool is_active
bool got_odometry_message
bool transform_ok
bool is_inside_range_limit